cultivation/scripts/flashcore/cli/main.py

```
@@ -0,0 +1,223 @@
+
+@export_app.command("md")
+def export_md(
+    output_dir: Optional[Path] = typer.Option(
+        None,
+        "--output-dir",
+        help="Directory to save exported Markdown files. Defaults to settings.export_dir",
+        file_okay=False,
+        dir_okay=True,
+        writable=True,
+        resolve_path=True,
+    ),
```

Ruff (0.11.9)

Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)